### PR TITLE
BUG: Fix bug with compute_maxwell_basis(..., int_order=0)

### PR DESCRIPTION
**Pull request by larsoner**
Fixes a small bug with `compute_maxwell_basis(info, int_order=0)` and adds doc requested by @cbrnr in #11559 for test_import_nesting

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -32,7 +32,7 @@ xlrd
 imageio>=2.6.1
 imageio-ffmpeg>=0.4.1
 traitlets
-pyvista>=0.32,!=0.35.2,!=0.38.0,!=0.38.1,!=0.38.2,!=0.38.3
+pyvista>=0.32,!=0.35.2,!=0.38.0,!=0.38.1,!=0.38.2,!=0.38.3,!=0.38.4
```
Comment (drammock) [inline requirements.txt:35]: this is getting ridiculous: 5 releases in a row are not working for us. Can we somehow upstream a test that uses MNE? cc @banesullivan

Comment (larsoner) [reply requirements.txt:35]: https://github.com/pyvista/pyvista/pull/4036 https://github.com/pyvista/pyvistaqt/pull/317

Comment (larsoner) [reply requirements.txt:35]: (the failure here ends up showing up as a doc build failure, though, so the integration test itself does not catch it)

Comment (banesullivan) [reply requirements.txt:35]: I just returned from vacation... will look into those issues to help resolve this.

**Comment by larsoner** [comment]
M1 test failure is probably just https://www.githubstatus.com/incidents/ybnn77s3lyf8 and I think we can ignore it

File: mne/tests/test_import_nesting.py [hunk 2]
```diff
@@ -5,6 +5,15 @@
...
 
+# To keep `import mne` time down, we nest most imports, including those of
+# required dependencies (e.g., pooch, tqdm, matplotlib). SciPy submodules
+# (esp. linalg) can take a while to load, so these imports must be nested, too.
+# NumPy is the exception -- it can be imported directly at the top of files.
+# N.B. that jinja2 is imported directly in mne/html_templates, so all imports
+# of mne.html_templates must be nested to achieve the jinja2 import being
+# nested during import of mne.
+#
+# This test ensures that we don't accidentally un-nest any of these imports.
```
Comment (larsoner) [inline mne/tests/test_import_nesting.py:16]: @cbrnr okay?

Comment (cbrnr) [reply mne/tests/test_import_nesting.py:16]: Great, thank you!

**Pull request merged**